### PR TITLE
Added social for DEV (https://dev.to/)

### PR DESCRIPTION
**Pull request by BaileyJM02**
The icon for DEV and url etc. have been added to the `social_media.yml` file and I’ve included an example within the `_config.yml` file. 

> The height of the icon is 22px as to align it with other icons on output.

### CODE REVIEW

**Review by thematchless — APPROVED** [review]
good idea 👍 tested and approved ✅

**Comment by BaileyJM02** [comment]
Any other reviewers?

File: _config.yml [hunk 1]
```diff
@@ -43,6 +43,7 @@ projects:
   # vk: your_username
   # website: http://your_website_url
   # youtube: your_username
+  # dev: your_username
```
Comment (pheeria) [inline _config.yml:46]: I think, all of the others are listed in alphabetical order.

File: _data/social_media.yml [hunk 2]
```diff
@@ -77,3 +77,8 @@ youtube:
   name: YouTube
   profile_url_prefix: https://www.youtube.com/
   icon_svg: '<svg height="20" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.17 13.6"><path d="M18.77 2.13A2.4 2.4 0 0 0 17.09.42C15.59 0 9.58 0 9.58 0a57.55 57.55 0 0 0-7.5.4A2.49 2.49 0 0 0 .39 2.13 26.27 26.27 0 0 0 0 6.8a26.15 26.15 0 0 0 .39 4.67 2.43 2.43 0 0 0 1.69 1.71c1.52.42 7.5.42 7.5.42a57.69 57.69 0 0 0 7.51-.4 2.4 2.4 0 0 0 1.68-1.71 25.63 25.63 0 0 0 .4-4.67 24 24 0 0 0-.4-4.69zM7.67 9.71V3.89l5 2.91z" fill="#959da5"/></svg>'
+
+dev:
```
Comment (pheeria) [inline _data/social_media.yml:81]: As mentioned above, the order is out of order :)

**Comment by pheeria** [comment]
You can also update README.md to include `dev`. 
I really like your changes!
To facilitate PR review for the maintainers, I'd suggest you add a screenshot of the new icon next to existing ones, because nobody can visualize the pure SVG :D

**Pull request closed without merging**